OsmAnd/src/net/osmand/plus/GpxSelectionHelper.java

```diff
@@ -54,7 +56,12 @@
 	private Map<GPXFile, Long> selectedGpxFilesBackUp = new java.util.HashMap<>();
 	private SavingTrackHelper savingTrackHelper;
 	private final static Log LOG = PlatformUtil.getLog(GpxSelectionHelper.class);
+	private SelectGpxTask selectGpxTask;
+	private SelectGpxTaskListener gpxTaskListener;
 
+	public void setGpxTaskListener(SelectGpxTaskListener gpxTaskListener) {
```

Comment:
No field is needed ! Fields should be created carefully and when it is not needed better to use parameter for method

Reply:
I know, but with this listener, we can get updates for necessary screens if update working in the background and it doesn`t update new screen.